modules/system/assets/ui/docs/select.md

```diff
@@ -43,6 +43,15 @@ Add the `select-no-search` CSS class to disable searching.
         </select>
     </div>
 
+### Dynamic option creation
+
+In addition to a prepopulated menu of options, Select2 can dynamically create new options from text input by the user in the search box. This feature is called "tagging". To enable tagging, set the `tags` option to `true`:
+
+    <select
+        class="form-control custom-select"
+        data-tags=true
```

Comment:
@meysammahfouzi 
```suggestion
        data-tags="true"
```

Reply:
Is this necessary? I test `data-tags=true` and it worked.

Reply:
@meysammahfouzi Our standards are to have quotes around attribute values.